sdk/resources/arm-resources/package.json

```diff
@@ -47,12 +60,37 @@
     "src/**/*.ts",
     "README.md",
     "rollup.config.js",
-    "tsconfig.json"
+    "tsconfig.json",
+    "review/*",
+    "CHANGELOG.md"
   ],
```

Comment:
@sarangan12  Let's change this to just
```
"files": [
    "dist/",
    "dist-esm/src/",
    "types/arm-resources.ts",
    "README.md",
    "LICENSE",
    "review/*",
    "CHANGELOG.md"
  ],
```

Reply:
@deyaaeldeen I think `LICENSE` should be chanegd to `LICENSE.txt` here.

Reply:
@dw511214992 it is the other way around, the license file should be named `LICENSE` but the code generator generates `LICENSE.txt` still. @sarangan12 can you please rename the generated license file to just `LICENSE`?

Reply:
we only can change it manally.

Reply:
This task will be completed along with PR https://github.com/Azure/autorest.typescript/pull/1097

Reply:
Code changes complete